tgui/packages/tgui/interfaces/NtosMessenger.js

```diff
@@ -0,0 +1,221 @@
+                    {message.outgoing ? (
+                      "(OUTGOING)"
+                    ) : (
+                      "(INCOMING)"
+                    )}
+                  </Box>
+                  {message.outgoing ? (
+                    <Box bold>
+                      {message.name + " (" + message.job + ")"}
+                    </Box>
+                  ) : (
+                    <Button transparent
+                      content={message.name + " (" + message.job + ")"}
+                      onClick={() => act('PDA_sendMessage', {
+                        name: message.name,
+                        job: message.job,
+                        ref: message.ref,
+                      })}
+                    />
+                  )}
+                </Section>
+                <Section mt={-1}>
+                  <Box italic>
+                    {message.contents}
+                  </Box>
+                  {!!message.photo && (
+                    <Box
+                      as="img"
+                      src={message.photo}
+                    />
+                  )}
+                </Section>
+              </Stack>
+            ))}
+          </Stack>
+        </NtosWindow.Content>
+      </NtosWindow>
+    );
+  }
+  return (
+    <NtosWindow width={600} height={800}>
+      <NtosWindow.Content>
+        <Stack vertical>
+          <Section fill textAlign="center">
+            <Box bold>
+              <Icon name="address-card" mr={1} />
+              SpaceMessenger V6.4.7
+            </Box>
+            <Box italic opacity={0.3}>
+              Bringing you spy-proof communications since 2467.
+            </Box>
+          </Section>
+        </Stack>
+        <Stack vertical>
+          <Section fill textAlign="center">
+            <Box>
+              <Button
+                icon="bell"
+                content={ringerStatus ? 'Ringer: On' : 'Ringer: Off'}
+                onClick={() => act('PDA_ringerStatus')}
+              />
+              <Button
+                icon="address-card"
+                content={sAndR ? "Send / Receive: On" : "Send / Receive: Off"}
+                onClick={() => act('PDA_sAndR')}
+              />
+              <Button
+                icon="bell"
+                content="Set Ringtone"
+                onClick={() => act('PDA_ringSet')}
+              />
+              <Button
+                icon="comment"
+                content="View Messages"
+                onClick={() => act('PDA_viewMessages')}
+              />
+              <Button
+                icon="sort"
+                content={sortByJob ? "Sort by: Job" : "Sort by: Name"}
+                onClick={() => act('PDA_changeSortStyle')}
+              />
+              {!!virus_attach && (
+                <Button
+                  icon="bug"
+                  color="bad"
+                  content={sending_virus ? "Attach Virus: Yes" : "Attach Virus: No"}
```

Comment:
I think that's how you do this in javascript (also 100% convert this to typescript)
```suggestion
                  content={`Attach Virus: ${sending_virus ? "Yes" : "No"}`}
```